src/app/core/teams/create-team/create-team.component.ts

```diff
@@ -127,7 +127,7 @@ export class CreateTeamComponent implements OnInit {
 		this.teamsService
 			.create(this.team, this?.teamAdmin?.userModel._id)
 			.pipe(
-				tap(() => this.authenticationService.reloadCurrentUser()),
+				switchMap(() => this.sessionService.reloadSession()),
```

Comment:
Just switching the call to `reloadSession` wasn't enough.  When using the `tap` operator, the returned Observable is never subscribed to, so the session is never reloaded.  Using `switchMap` ensures that the returned observable is subscribed to properly.